add fix for EC-Earth further_info_url

ellesmith88 opened the pull request:
For PR https://github.com/roocs/dachar/issues/96

Added a fix for the `further_info_url` attribute. To make this more straightforward, I added a fix that every decadal dataset will use but only EC-Earth3 datasets with `"none"` in the `further_info_url` have this attribute changed. 

This means that all the decadal fixes can be created from the same template.  (Corresponding updated template here: https://github.com/roocs/dachar/blob/decadal_derive/generate_decadal_fix.py) If you think this should be separate I can remove it and add it into the template for the specific datasets only.

agstephens commented:
Hi @ellesmith88, I can see the sense of your approach to push all the decadal data through this fix. However, I think one of the issues that we want to manage with all our fixes is that they can be _dataset-specific_. So, I think it is good practice for us to be able to look at the _fix-store_ and find out that the fix for the `further_info_url` was only applied to certain datasets. If it would take a long time to make this change then we can leave it for now, but if it is relatively easy then please modify it so that only the relevant datasets will be tagged as needing this fix. Thanks

ellesmith88 commented:
@agstephens I've updated this now, in the test index only one has the further info url fix. 

I've updated the `generate_decadal_fix` template in dachar to add this extra fix in only when needed (https://github.com/roocs/dachar/blob/decadal_derive/generate_decadal_fix.py). I used the list of datasets to add it in (see https://github.com/roocs/dachar/blob/01c041993bb227d1850db4ecfd0b3e0047cf7eb0/generate_decadal_fix.py#L184-L187), there may be a better way but I couldn't think how else to do it.
